yarn.lock

```diff
@@ -3824,6 +3824,13 @@
   resolved "https://registry.yarnpkg.com/@storybook/theming/-/theming-8.4.7.tgz#c308f6a883999bd35e87826738ab8a76515932b5"
   integrity sha512-99rgLEjf7iwfSEmdqlHkSG3AyLcK0sfExcr0jnc6rLiAkBhzuIsvcHjjUwkR210SOCgXqBPW0ZA6uhnuyppHLw==
 
+"@swc/helpers@^0.5.15":
```

Comment:
Without changes in `package.json` all those package requirements will be lost after a fresh `yarn install`.